en/flight_controller/crazyflie2.md

```diff
@@ -117,10 +117,9 @@ obtain the latest QGC source code (master) and build it.)
 
 ## Hardware Setup
 
-Uptill now we have been able to fly crazyflie with precise control in Stabilised and Altitude modes. 
+Crazyflie is able to fly with precise control in Stabilised, Altitude and Position modes. 
 
-To fly in Altitude mode:
-* You will need the [Z-ranger deck](https://store.bitcraze.io/collections/decks/products/z-ranger-deck) or the distance sensor integrated in the [Flow deck](https://store.bitcraze.io/collections/decks/products/flow-deck).
+* You will need the [Z-ranger deck](https://store.bitcraze.io/collections/decks/products/z-ranger-deck) to fly in Altitude mode. If you also want to fly in the Position mode, it is recommended you buy the [Flow deck](https://store.bitcraze.io/collections/decks/products/flow-deck) which also has the Z-ranger sensor integrated in it.
```

Comment:
Generally speaking, first instance of each mode link to the Mode document - e.g. [Altitude mode](../flight_modes/altitude_mc.md). Subsequently it is quite good to mark up the mode in italic: *Altitude mode*.